package.json

```diff
@@ -50,7 +50,8 @@
     "ts-node": "^10.7.0",
     "tsconfig-paths": "^3.14.1",
     "typescript": "^4.3.5",
-    "yargs": "^17.4.0"
+    "yargs": "^17.4.0",
+    "yarn-audit-fix": "^9.3.10"
```

Comment:
Once you've run `yarn-audit-fix`, it has changed your `lock` file. Is there any point of keeping this dependency around?

Reply:
Fair enough! Removed in 1e9c4202ac2c8ef65f2ca4482db6d742f73efe5b